ocp4-logging/galaxy-requirements.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
why have two requirements files? couldn't they be merged? any reason?

Reply:
The two requirements files serve two different purposes. One is for bringing in the applier so that it can be used and the other is bringing in requirements use in the post-step by the applier itself. I'm not sure that there's a technical reason behind splitting them up -- but my preference has always been to seperate the two to make it clear how they're being used.

Reply:
Agreed on keeping the galaxy requirements file to just what's needed to avoid extra overhead. However, I'm questioning the use/need for a requirements file for the applier itself as it's just another way of running `git clone`  - i.e.: why wrap it in another command?

Reply:
Not sure I agree with the resolution. It’s certainly not an OpenShift-applier issue, but rather a topic to be addressed here as this PR introduces the steps/procedure, no?

Reply:
So in this specific instance -- I'm using the requirements file for the `openshift-applier` to make sure that we have an appropriate version that will work with this. I believe only versions >= 2.1 will work with OCP4, so this is just an easy step to make sure that we avoid problems with versions lower than that. Happy to remove it if everyone else thinks this is a non-issue.

Reply:
OK, yeah, I see your point, and can live with that. Alternatively, instead of the `ansible-galaxy` command, it would just be `git clone -b v2.1.2 https://github.com/redhat-cop/openshift-applier` ... but I suppose there's slightly a bigger risk of people missing the `-b` part.